src/DataTool.php

```diff
@@ -181,7 +168,32 @@ public function generateData()
             }
         }
     }
-    
+
+    /**
+     * @param $field string - stores field name from fields
+     * @param $fieldDef array - stores data definition for proper field
+     */
+    public function generateFieldData($field, $fieldDef)
```

Comment:
Please make it private so that we are confident that nobody is calling this method outside this class.